.eslintignore

```diff
@@ -7,5 +7,6 @@ doc/api/*
 doc/examples/jest_react/*.js
 
 lib/core/imports/*.js
+lib/core/utils/uuid.js
```

Comment:
Did you mean to include this?

Reply:
I did @WilcoFiers. It felt weird to update a file that I don't think we wrote (judging by the comment at the top of `uuid.js`. Eventually when all of these codemods are merged the last PR will turn on the eslint rule `no-var` and unless we either update this file or ignore it, this file will throw an eslint warning.

Reply:
@gaiety-deque Merge conflict, can you update?